Dockerfile

```diff
@@ -55,10 +21,4 @@ COPY sources /app/sources/
 COPY adapters /app/adapters/
 COPY migrations /app/migrations/
 
-#############################
-# entrypoint
-#
-RUN source $NVM_DIR/nvm.sh
-ADD .build_scripts/entrypoint.sh /
-RUN chmod +x /entrypoint.sh
-ENTRYPOINT ["/entrypoint.sh"]
+CMD ["npm", "start"]
```

Comment:
(this might break the ECS task definition, please take this as a suggestion)

Perhaps we can set `npm` as the ENTRYPOINT, and set `start` as the default command like so:

```
ENTRYPOINT ["/usr/local/bin/npm"]
CMD ["start"]
```

This way people can only run `npm` commands, but can override the default command if they wish.

For example, this will run the default `start` command:
```
docker run --rm openaq_fetch:latest
```

While these will override the `start` command:
```
docker run --rm openaq_fetch:latest test 
docker run --rm openaq_fetch:latest start -- --dryrun -s "Beijing US Embassy"
```

Reply:
I really like this suggestion but I couldn't get it to work, I might have just been doing something wrong. I'll definitely keep it in mind and hopefully we can implement it in future changes!